domains/jake.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Jake",
+    "email": "hi.cybot@gmail.com",
+    "note": "That email address is not my main email, to make it faster reply to me with a message."
```

Comment:
wdym "reply"?

Reply:
as for replying in the pull request thread

Reply:
we can't have your pr info, so better link more social accounts :)

Reply:
Added Discord, that's all I can add for now xD

Reply:
mmlol feel free to join our discord to receive notification about the service, also we'll host community event there!

Reply:
wait is the file changed? i can't see the file being modified.

Reply:
Weird.. try clicking here https://github.com/Aceeeeeeeeeee/register/blob/Aceeeeeeeeeee-patch-1/domains/jake.json

Reply:
![image](https://user-images.githubusercontent.com/70831061/162598040-8f949ea4-1242-4b8d-b0dc-ca00c615db15.png)
it is in the `Aceeee-patch-1`, but you are asking us to merge the content in the `main` branch.

Reply:
My bad, I have registered in github a long time ago and still didn't manage to learn the basics lmfao

Reply:
I deeply apologise if you find me confusing 😅👍

Reply:
mm

Reply:
*anything else??*